### PR TITLE
AS7-6453 and README.md clean-up

### PR DESCRIPTION
**Pull request by kwart**

### CODE REVIEW

File: ejb-security-interceptors/src/main/resources/META-INF/jboss-ejb3.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
   -->
-<jboss xmlns="http://www.jboss.com/xml/ns/javaee"
+<ejb-jar xmlns="http://www.jboss.com/xml/ns/javaee"
```
Comment (jaikiran) [inline ejb-security-interceptors/src/main/resources/META-INF/jboss-ejb3.xml]: This isn't right. Why was this change required?

**Comment by sgilda** [comment]
Bug 911483 was entered for this. I will hold off doing anything until Jaikiran's question is answered.

**Comment by sgilda** [comment]
Please remove the metadata tag edits and the carriage returns that were added to all the text. :-)

File: ejb-security-interceptors/src/main/resources/META-INF/jboss-ejb3.xml (hunk 1 above)
Comment (kwart) [inline ejb-security-interceptors/src/main/resources/META-INF/jboss-ejb3.xml]: It's weird. I had issues with creating SwitchIdentityTestCase in AS7 testsuite and I was convinced it's related to this. But I'm not able to reproduce it now. So I'll change it back. Sorry.

**Comment by kwart** [comment]
I hope the `README.md` is fixed now.

**Comment by sgilda** [comment]
@kwart : I noticed you changed the formatting for all the top level sections in the file. It's not really a big deal except that we're trying to make sure all the quickstarts use the same formatting, using the template/README.md file as a guide, at least for common top level sections. This will allow us to more easily parse and scan to automate validation of things we expect to see in the file. If we change them in this file, we would need to update them in all the other README files to keep them consistent. I actually prefer the `#` symbols, but I'm  too busy (and lazy) to change all the files at this point.

Would you mind changing the top level section headers, including the first line, back to the original format? I'm not concerned about the subsections.

**Comment by kwart** [comment]
should be returned now.

**Comment by sgilda** [comment]
The README changes are good. 

@pmuir: Do you want to review the code or should I ask Darran or Jaikiran to review it?

@kwart: If I get the OK, I assume these changes need to go into the EAP 6.1 quickstarts too?

**Comment by kwart** [comment]
Yes, it should be merged to the EAP 6.1 too .

**Comment by sgilda** [comment]
@kwart, @darranl: While testing this for EAP 6.1, I noticed that under "Modify the Server Configuration using the JBoss CLI Tool", it tells you to start the server with `JBOSS_HOME_SERVER_1/bin/standalone.sh -c standalone-full.xml`. However, under "Start JBoss Enterprise Application Platform 6.1 or JBoss AS 7.2", it says to start the server using `JBOSS_HOME/bin/standalone.sh` without the argument. Does this quickstart require the full profile? I assume it does?

**Comment by kwart** [comment]
@sgilda: No. AFAIK the default profile should be fine too.

**Comment by sgilda** [comment]
@kwart: When you updated the README file, you added the full profile argument to the command to start the server. Later, when you start the server, you don't pass the profile that contains the server configurations. Don't they have to be in sync?

**Comment by darranl** [comment]
Yes there is nothing HA specific in this example, I think standalone-full.xml crept in copying the CLI commands from a previous quick start.

**Comment by sgilda** [comment]
Merged.
